src/components/interface/filter/index.jsx

```diff
@@ -40,11 +41,11 @@ class Filter extends React.Component {
   }
 
   componentDidUpdate(prevProps, prevState) {
-    const { filtered: prevFiltered } = prevState;
-    const { filtered: currFiltered } = this.state;
+    const { filtered: prevFiltered, search: prevSearch } = prevState;
+    const { filtered: currFiltered, search: currSearch } = this.state;
     const { handleFilters } = this.props;
-    if (!isEqual(prevFiltered, currFiltered)) {
-      handleFilters(currFiltered);
+    if (!isEqual(prevFiltered, currFiltered) || prevSearch !== currSearch) {
```

Comment:
*apparantly* you could actually use !isEqual() for prevSearch and currSearch, it looks like it covers comparing between String objects.